threadpoolctl.py

```diff
@@ -73,28 +75,33 @@ class _dl_phdr_info(ctypes.Structure):
     "OpenBLASController": {
         "user_api": "blas",
         "internal_api": "openblas",
-        "filename_prefixes": ("libopenblas",),
+        "filename_prefixes": ("libopenblas", "libblas"),
+        "check_funcs": ("openblas_get_num_threads", "openblas_get_num_threads64_"),
```

Comment:
Maybe `check_symbols` would make it easier to understand the purpose of this key in the spec?